Net/include/Poco/Net/SocketDefs.h

```diff
@@ -26,7 +26,9 @@
 
 #if defined(POCO_OS_FAMILY_WINDOWS)
 	#include "Poco/UnWindows.h"
-	#define FD_SETSIZE 1024 // increase as needed
+	#ifndef FD_SETSIZE
+		#define FD_SETSIZE 1024 // increase as needed
```

Comment:
I quickly read about this and it might be that with MinGW default is 64. Would this be problem?

Reply:
Yes it is defined to 64. I don't know about the implications.

Reply:
Ok. Maybe we just go with Windows default here.